lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -97,6 +88,22 @@ class GoogleCloudOutput < BufferedOutput
     #   }
     config_param :label_map, :hash, :default => nil
 
+    # Credentials and Authorization
+    #
+    # Method 1: Use Application Default Credentials / GCE service account
+    #
+    # This method makes use of the application default credentials. These
+    # are provided by GCE, or through the the 'gcloud auth' command.
+    # This requires no parameters, though it may read credentials
+    # located by the environment variable GOOGLE_APPLICATION_CREDENTIALS.
+    #
+    # Method 2: Use Service account keys.
+    #
+    # service_credentials_path is a path to a service account key file,
+    # in json format, which can be acquired from the Google Cloud Developers
+    # Console. When set, the other fields are unnecessary.
+    config_param :service_credentials_path, :string, :default => nil
```

Comment:
When I did the ADC support I convinced myself it was better not to have this in the fluentd config.  Ideally the user will put the credentials in one of the well-known directories, and failing that they can use the environment variable.  Providing another way to do it seems like an unnecessary complication.